websites/A/AnimesGG/presence.ts

```diff
@@ -0,0 +1,139 @@
+        video = document.querySelector(
+          "#video > div > div > video"
+        ) as HTMLMediaElement,
+        timestamps = presence.getTimestampsfromMedia(video);
+      data.details = "Assitindo um anime!";
+      data.state = `Vendo ${animeTitleWatching}`;
+      data.smallImageKey = "play";
+      if (video.paused) {
+        delete data.endTimestamp;
+        delete data.startTimestamp;
+        data.smallImageKey = "pause";
+      }
+      [data.startTimestamp, data.endTimestamp] = timestamps;
+      data.buttons = [
+        {
+          label: "Asistir tambem",
+          url: `https://animes.gg/${page}`
+        }
+      ];
+    } else {
+      const animeTitleAbout = document.querySelector(
+          ".ani_titulo > h1"
+        ).innerHTML,
+        animeTitleFull = document.querySelector(
+          ".ani_titulo_original"
+        ).innerHTML,
+        showFullName = await presence.getSetting("fullName");
+      data.details = "Vendo sobre um anime.";
+      if (showFullName === true && animeTitleFull !== undefined && animeTitleFull !== null && animeTitleFull !== "") 
+        data.state = `Vendo sobre ${animeTitleFull}`;
+      else 
+        data.state = `Vendo sobre ${animeTitleAbout}`;
+    }
+  } else if (page.startsWith("/usuario")) {
+    const username = document.querySelector(
+      ".profile-name"
+    ).innerHTML;
+    data.details = "Vendo a a pagina de um usuario";
+    data.state = `Vendo a pagina do ${username}`;
+  } else if (page.startsWith("/filme")) {
+    const animeTitleAbout = document.querySelector(
+        ".ani_titulo > h1"
+      ).innerHTML,
+      animeTitleFull = document.querySelector(
+        ".ani_titulo_original"
+      ).innerHTML,
+      showFullName = await presence.getSetting("fullName"),
+      video = document.querySelector(
+        "#video > div > div > video"
+      ) as HTMLMediaElement,
+      timestamps = presence.getTimestampsfromMedia(video);
+    data.details = "Assitindo um filme de anime";
+    if (showFullName === true && animeTitleFull !== undefined && animeTitleFull !== null && animeTitleFull !== "") 
+      data.state = `Vendo sobre ${animeTitleFull}`;
+    else 
+      data.state = `Vendo sobre ${animeTitleAbout}`;
+    [data.startTimestamp, data.endTimestamp] = timestamps;
+    data.buttons = [
+      {
+        label: "Asistir tambem",
+        url: `https://animes.gg/${page}`
+      }
+    ];
+    data.smallImageKey = "play";
+    if (video.paused) {
+      delete data.endTimestamp;
+      delete data.startTimestamp;
+      data.smallImageKey = "pause";
+    }
+  } else if (page.startsWith("/genero")) {
+    data.details = "Procurando um genero de anime";
+    data.state = `Procurando ${page.substring("/genero/".length).slice(0, -1)}`;
+  } else if (page.startsWith("/lancamentos")) {
+    data.details = "Vendo os ultimos lancamentos no site.";
+    data.state = "Vendo...";
+  }
+
+  if (data.details && data.state)
+    presence.setActivity(data);
```

Comment:
```suggestion
  if (data.details && data.state) presence.setActivity(data);
  else presence.setActivity();
```

Set the app icon too if you haven't already